packages/graphback-datasync/src/providers/DataSyncConflictProvider.ts

```diff
@@ -3,7 +3,7 @@ import { DataSyncModelConflictConfig, DataSyncFieldNames, ConflictMetadata, Conf
 import { MongoDeltaSource } from '../deltaSource';
 import { DataSyncMongoDBDataProvider } from './DatasyncMongoDBDataProvider';
 
-export const MAX_RETRIES = 3;
+export const MAX_RETRIES = process.env.CONFLICT_RESOLUTION_MAX_RETRIES ? process.env.CONFLICT_RESOLUTION_MAX_RETRIES : 3;
```

Comment:
What you had was correct but this was a shorter way to do it:

```suggestion
export const MAX_RETRIES = process.env.CONFLICT_RESOLUTION_MAX_RETRIES || 3;
```